<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_all">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_one">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 20pt &quot;System&quot;;</string>
    </property>
    <property name="text">
     <string>bitset&lt;64&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="do_one">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:7px;
font: 700 20pt &quot;System&quot;;
color:rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>do</string>
    </property>
   </widget>
   <widget class="QPushButton" name="do_all">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:7px;
font: 700 20pt &quot;System&quot;;
color:rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>do</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(255, 255, 255);
font: 700 20pt &quot;System&quot;;
color: rgb(0, 0, 255)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="info">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
